content-repositories.json

```diff
@@ -4,7 +4,7 @@
   { "kind": "github", "project": "rackerlabs/docs-quickstart" },
   { "kind": "github", "project": "rackerlabs/rpc-vmware" },
   { "kind": "github", "project": "rackerlabs/docs-redhat-osp" },
-  { "kind": "github", "project": "rackerlabs/docs-rpc", "branches": ["master", "v10", "v11", "v12", "v13", "v14"] },
+  { "kind": "github", "project": "rcbops/docs-rpc", "branches": ["master", "v10", "v11", "v12", "v13", "v14"] },
```

Comment:
didn't we decide to keep the `rcbops/privatecloud-docs` name ?
also, we don't build `v10`, `v11`